hummingbird/ml/operator_converters/sklearn/gbdt.py

```diff
@@ -18,7 +18,7 @@
 
 def _get_n_features(model):
     try:
-        return model.n_features_
+        return model.n_features_in_
     except AttributeError:
         # HistGradientBoosting
         return model._n_features
```

Comment:
For HistGradientBoosting should we also use `n_features_in_`?

Reply:
In the official docs, there's `n_features_in_` in `HistGradientBoosting` but not `n_features_`.
I think we should use `n_features_in_` for `HistGradientBoosting`.

[sklearn.ensemble.HistGradientBoostingClassifier — scikit-learn 1.1.1 documentation](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingClassifier.html)
[sklearn.ensemble.HistGradientBoostingRegressor — scikit-learn 1.1.1 documentation](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingRegressor.html)

Reply:
Ok, let's remove the `except` part and use `n_features_in_` everywhere?

Reply:
Sounds good. To keep the PR small, it's good to work on another branch.
I've opened a new issue #606 to track that.